mw-webapp/src/utils/auth/writeNewUserData.ts

```diff
@@ -0,0 +1,17 @@
+import {ref, set} from "firebase/database";
+import {db} from "src/firebase";
+
+interface UserData {
+  userId: string
+  userEmail: string | null
+  userName: string | null
+}
+
+export const writeNewUserData = ({userId, userEmail, userName}: UserData) => {
+  const usersListRef = ref(db, "/users/" + userId);
```

Comment:
it is very hard to managed widespreaded strings in enterprise projects. move to service

Reply:
I remove destructing if you mean it